packages/instant/src/redux/async_data.ts

```diff
@@ -99,7 +99,12 @@ export const asyncData = {
                 dispatch,
                 selectedAsset as ERC20Asset,
                 selectedAssetUnitAmount,
-                { setPending: !options.updateSilently, dispatchErrors: !options.updateSilently, affiliateInfo },
+                {
+                    setPending: !options.updateSilently,
+                    dispatchErrors: !options.updateSilently,
+                    fetchedVia: options.fetchedVia,
```

Comment:
nit: it doesn't seem like this is an optional parameter right? There isn't a sensible default value.